packages/block-library/src/post-content/index.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-content` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-content` block on the server.
+ *
+ * @return string Returns the filtered post content of the current post.
+ */
+function render_block_core_post_content() {
+	rewind_posts();
+	the_post();
```

Comment:
Should the post not be provided as an argument?

Reply:
Not really, this block renders whatever post its context dictates.

We do need to add support for pages with multiple posts/post blocks.